src/controllers/user.controllers.js

```diff
@@ -37,5 +39,8 @@ export const userLogin = async (data, response) => {
   const validPass = await bcrypt.compare(data.password, user.password);
   if (!validPass) return response.status(StatusCodes.BAD_REQUEST).send('Email or password is wrong');
 
+  const token = jsonwebtoken.sign({_id:  user._id}, process.env.TOKEN_SECRET)
+  response.header('auth-token', token);
```

Comment:
Shouldn't token be sent in response's body?

Reply:
In PR user-logout-endpoint i set thid token to cookies like this "response.cookie('auth', token, { maxAge: 900000, httpOnly: true });"